test/service/__init__.py

```diff
@@ -92,3 +97,20 @@ def _fill_index(cls, num_docs=1000):
                                                       aggregate=True),
                            doc_type='meta',
                            refresh='wait_for')
+
+
+class DSSTestCase(TestCase):
+    """
+    This TestCase should be used as a mixin to provide DSS related environment
+    variables to testcases that would break if the variables were not set.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls._catalog_mock = mock.patch.dict(os.environ,
+                                            AZUL_DSS_ENDPOINT='https://dss.data.humancellatlas.org/v1')
+        cls._catalog_mock.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._catalog_mock.stop()
```

Comment:
![image](https://user-images.githubusercontent.com/5143256/102833566-4e2aec00-43a6-11eb-8c1c-06b051072ca9.png)

Reply:
the name needs to be changed too. >.>